.gitignore

```
@@ -4,6 +4,9 @@
 .classpath
 bin/
 
+# IntelliJ Idea
+.idea
```

```suggestion
/.idea/
```

Limit to a folder in the root of the repo.